### ISSUE TITLE
Firefox: custom CSS code randomly resets to default when Firefox starts

### ISSUE THREAD

**Issue by Noexpert** [philc/vimium#2913]
Issue as above.
I've not been able to link it to anything specific.

Thanks

**Comment by innaterebel** [comment]
You mean the code you entered in the Vimium settings actually got reset (to default codes? to blank? while all other settings retained?), 
or just doesn't take effect sometimes?

**Comment by Noexpert** [comment]
Hi, I mean it resets to the default Vimium code.
This happens on two different machines.

Thanks

**Comment by smblott-github** [comment]
@Noexpert ... And all of the other settings are retained?

**Comment by Noexpert** [comment]
Yes, they seem to be, although to be fair, I don't think I've changed much else.

**Comment by Noexpert** [comment]
To add to my comments above: I've noticed that this does not necessarily happen when restarting. In fact it happens during the same session and at random points in time.

So on one tab it's fine and on the next one it suddenly changes to default.

**Comment by gerleo** [comment]
I've had a similar issue with the "Require Enter when filtering hints" option. I think it's being reset to default (checked) when Firefox restarts.

**Comment by smblott-github** [comment]
See #2929 (possibly related).

> All of the saved settings are lost when you clear the cache by selecting everything in the "Clear All History" window.
